repository-data/development/src/main/resources/hcm-content/content/documents/lks/case-studies.yaml

```diff
@@ -111,3 +111,91 @@
           jcr:encoding: UTF-8
           jcr:lastModified: 2021-05-11T13:06:13.939+01:00
           jcr:mimeType: application/pdf
+  /search-case-study:
```

Comment:
Could we reload `/content/documents/lks/case-studies` ?

Reply:
Added